tests/Issue.py

```diff
@@ -174,3 +175,29 @@ def testCreateReaction(self):
 
         self.assertEqual(reaction.id, 16917472)
         self.assertEqual(reaction.content, "hooray")
+
+    def testGetTimeline(self):
+        expected_events = {'referenced', 'cross-referenced', 'locked', 'unlocked', 'closed', 'assigned', 'commented', 'subscribed', 'labeled'}
+        events = self.issue.get_timeline()
+        for event in events:
+            print('EVENT: ', event)
```

Comment:
Useful for debugging, not so useful for test runs. :-)

Reply:
Hah. Whoops. Thanks.